config/initializers/schedule_parameters.rb

```diff
@@ -0,0 +1,6 @@
+#sanitize the OSEM_SCHEUDLE_CELL_SIZE to be used for EventType::LENGTH_STEP
```

Comment:
ok, here is my position on that and why I did that:
1 - there should be some sanity checks since we don't know, if the one who will be setting up the software has spent some time reading the docs etc. there IS a room for human error. Otherwise it may be done in the model itself
2 - all config stuff initialization should be done in the initializers, that's why they are for, this is a common way of doing it. And I am not sure if it is a responsibility of the model to do such sanity checks